docs/intro.rst

```diff
@@ -331,6 +331,17 @@ accordingly:
 
 1. The Web Services API must be enabled on the HMC.
 
+   You can do that in the HMC GUI by selecting "HMC Management" in the left pane,
```

Comment:
If *GUI* is the established term in the zhmcclient documentation, this is fine.
Otherwise, it might be worth asking the END team about the preferred terminology.

Reply:
Started Slack discussion with Larry and you on this.

Reply:
Larry is OK with the term "GUI".